app/assets/javascripts/directives/validation/validation_status.js

```diff
@@ -1,16 +1,21 @@
 ManageIQ.angular.app.directive('validationStatus', ['$rootScope', function($rootScope) {
   return {
     require: 'ngModel',
-    link: function (scope, elem, attrs, ctrl) {
-      ctrl.$validators.validationRequired = function (modelValue, viewValue) {
-        if (viewValue !== undefined && viewValue === true) {
+    link: function(scope, _elem, attrs, ctrl) {
+      ctrl.$validators.validationRequired = function(_modelValue, viewValue) {
+        if (viewValue === true) {
+          // NOTE: this is for auth-credentials component
+          if (! attrs.mainScope) {
+            var vm = scope.$parent.vm;
```

Comment:
It is generally not a good idea to assume `$parent.vm`.

`parent.vm` should be passed in as a parameter (via attributes) from the haml.

Reply:
Trouble here is, that $parent is not pointing towards the *formController*, but the component itself. The `postvalidationModelRegistry()`, is passed as a property to `AuthCredentials` component, the only problem is accessing it. I will look into it further, but from what i know, the isolate scope inside directive might be way to go.

Reply:
@Hyperkid123 Any chance we can update `validationStatus` and just pass in the right method from outside?